docs/registry/configure.md

```diff
@@ -403,20 +415,6 @@ To add a local module on your machine:
 
 1. Click **Create** to create the modular resource provided by the local module.
 
-You can also add the module directly, without first adding its modular component or service:
```

Comment:
Agreed with this deletion on principle, but I think it helps to retain awareness with a single sentence: some users will want to add a local module, and then 4 instances of the motor component it provides for their four-wheel machine. What about:

> Once you've added your local module using steps 1-6, you can repeat steps 7-10 to add as many additional instances of your modular resource as you need.

Or similar.